docs/how_to_guides/logs.md

```diff
@@ -4,7 +4,7 @@ All `Guard` calls are logged internally, and can be accessed via the guard histo
 
 ## Accessing logs via `Guard.history`
 
-`history` is an attribute of the `Guard` class. It implements a standard `Stack` interface with a few extra helper methods and properties.  For more information on our `Stack` implementation see the [Helper Classes](/docs/api_reference_markdown/helper_classes) page.
+`history` is an attribute of the `Guard` class. It implements a standard `Stack` interface with a few extra helper methods and properties.  For more information on our `Stack` implementation see the [Helper Classes](/docs/api_reference/helper_classes) page.
```

Comment:
We removed the docs on helper classes, so this link doesn't lead to any API refs currently. Can you remove this entire sentence?

`For more information on our `Stack` implementation see the [Helper Classes](/docs/api_reference/helper_classes) page.`